Doc/library/tkinter.scrolledtext.rst

```diff
@@ -25,12 +25,14 @@ be used directly to achieve most normal geometry management behavior.
 Should more specific control be necessary, the following attributes are
 available:
 
+.. class:: ScrolledText
```

Comment:
What is the signature of the init method?  I think some other signatures are missing also.